README.md

```diff
@@ -1,5 +1,13 @@
 # Status-desktop
 
-Desktop client for the [Status Network](https://statusnetwork.com/) built with [Nim](https://nim-lang.org/) and [Qt](https://www.qt.io/)
+Desktop client for the [Status Network](https://status.app/) built with [Nim](https://nim-lang.org/) and [Qt](https://www.qt.io/)
```

Comment:
I wonder if `Status Network` is a correct terminology now. We use this for a different thing.